README.md

```diff
@@ -34,37 +32,58 @@ Sends a deployment complete message to a Slack channel when [Trellis](https://gi
 - Valid Slack Webhook URL
 
 ## Screenshots
+
 ![Deployment Screenshot](deployment-screenshot.png)
 
 ## Installation
 
-Add this role to `requirements.yml`:
+Add this role to `requirements.yml` or `galaxy.yml` in newer versions of Trellis:
+
 ```yaml
-# requirements.yml
-- src: https://github.com/ItinerisLtd/trellis-slack-webhook-notify-during-deploy
-  version: 0.1.0 # Check for latest version!
+# requirements.yml / galaxy.yml
+- name: slack-notify
+  src: https://github.com/ItinerisLtd/trellis-slack-webhook-notify-during-deploy
+  version: 0.3.0 # Check for latest version!
```

Comment:
@mike-sheppard if you could resolve the merge conflict, remove `name` as per #12 and resolve the change in the vendor paths then we can get this merged and a new release published :)
Sorry it's taken so long to get around to this.

Reply:
aha just seen this:
> resolve the change in the vendor paths...

will sort this out shortly